### PR TITLE
Update DBRP examples

### LINKED ISSUE #1851: DBRP: Replace raw API calls with CLI-based examples
_Describe the issue here._
Now that the CLI has been enhanced to support management of the DBRP mappings, I'd suggest we relegate the API calls to the reference guide.

Drive all examples through the DBRP CLI commands.

##### Relevant URLs
https://docs.influxdata.com/influxdb/v2.0/query-data/influxql/#map-unmapped-buckets

https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/dbrp/
We should reference the CLI commands from this page as well ^

##### What products and version are you using?
2.0.2 OSS

### LINKED ISSUE #1883: Cloud-specific DBRP API examples
Currently, the DBRP API differs between OSS and Cloud. We need to provide curl examples for interacting with the DBRP API in Cloud until the implementations are fixed and aligned.

Related to influxdata/idpe#9267
Related to #1851

##### Relevant URLs
- https://docs.influxdata.com/influxdb/v2.0/query-data/influxql/#map-unmapped-buckets
- https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/dbrp/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1851 
Closes #1883 

This is a stop-gap change. The DBRP API implementations are currently different. Because of the differences, the `influx v1 dbrp` CLI will not work with cloud. This PR adds CLI examples to the DBRP mapping sections of the **Query with InfluxQL** doc **in OSS**. This updates the `curl` examples in the Cloud version of this doc to accurately reflect the Cloud implementation of the DBRP API.

Once the Cloud DBRP API is fixed and aligned with OSS, we can just use `{{< duplicate-oss >}}` to populate the Cloud content. That work is being tracked here: https://github.com/influxdata/idpe/issues/9267.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Review by timhallinflux — APPROVED** [review]

**Review by danxmoran — APPROVED** [review]
LGTM

File: content/influxdb/v2.0/query-data/influxql.md [hunk 1]
```diff
@@ -51,48 +88,91 @@ curl --request GET \
...
+Include the following in your request:
 
-- InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
-  _(See [compatibility API authentication](/influxdb/v2.0/reference/api/influxdb-1x/#authentication))_
-- **db query parameter**: 1.x database to query
-- **rp query parameter**: 1.x retention policy to query; if no retention policy is specified, InfluxDB uses the default retention policy for the specified database.
-- **q query parameter**: InfluxQL query
+- **Request method:** `GET`
+- **Headers:**
+  - **Authorization:** _See [compatibility API authentication](/influxdb/v2.0/reference/api/influxdb-1x/#authentication)
```
Comment (kelseiv) [inline content/influxdb/v2.0/query-data/influxql.md:171]: tion)_

File: content/influxdb/cloud/query-data/influxql.md [hunk 2]
```diff
@@ -15,4 +15,153 @@ related:
...
+  --header "Authorization: Token YourAuthToken" \
+  --data-urlencode "q=SELECT used_percent FROM example-db.example-rp.example-measurement WHERE host=host1"
+```
+
+By default, the `/query` compatibility endpoint returns results in **JSON**.
+To return results as **CSV**, include the `Accept: application/csv` header.
+
+## InfluxQL support
+
+InfluxDB Cloud and InfluxDB OSS 2.0 support InfluxQL **read-only** queries. See supported and unsupported queries below.
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:133]: InfluxDB supports

File: content/influxdb/cloud/query-data/influxql.md [hunk 3]
```diff
@@ -15,4 +15,153 @@ related:
...
+
+1. [Verify buckets have a mapping](#verify-buckets-have-a-mapping).
+2. [Map unmapped buckets](#map-unmapped-buckets).
+3. [Query a mapped bucket with InfluxQL](#query-a-mapped-bucket-with-influxql).
+
+## Verify buckets have a mapping
+
+{{% note %}}
+When writing to an InfluxDB Cloud bucket using the `/write` 1.x compatibility API,
+InfluxDB Cloud automatically creates a DBRP mapping for the bucket.
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:33]: to be sure, does `/write` to Cloud still create a DBRP mapping?

File: content/influxdb/cloud/query-data/influxql.md [hunk 4]
```diff
@@ -15,4 +15,153 @@ related:
...
+For more information, see [Database and retention policy mapping](/influxdb/cloud/reference/api/influxdb-1x/dbrp/).
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:34]: clarify on Database and retention policy mapping page (if there is distinction)

File: content/influxdb/cloud/query-data/influxql.md [hunk 5]
```diff
@@ -15,4 +15,153 @@ related:
...
+If you **do not find a DBRP mapping for a bucket**, complete the next procedure to map the unmapped bucket.
+_For more information on the DBRP mapping API, see the [`/api/v2/dbrps` endpoint documentation](/influxdb/cloud/api/#tag/DBRPs)._
+
+## Map unmapped buckets
+Use the [`/api/v2/dbrps` API endpoint](/influxdb/cloud/api/#operation/PostDBRP) to create a new DBRP mapping.
+Include the following:
+
+- **Request method:** `POST`
+- **Headers:**
+  - **Authorization:** `Token` schema with your InfluxDB [autorization token](/influxdb/cloud/security/tokens/)
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:77]: typo

File: content/influxdb/cloud/query-data/influxql.md [hunk 6]
```diff
@@ -15,4 +15,153 @@ related:
...
+  - **Content-type:** `application/json`
+- **Request body:** JSON object with the following fields:
+  - **bucket_id:** [bucket ID](/influxdb/cloud/organizations/buckets/view-buckets/) <span class="req">Required</span>
+  - **database:** database name <span class="req">Required</span>
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:81]: (Required) **database:**

File: content/influxdb/cloud/query-data/influxql.md [hunk 7]
```diff
@@ -15,4 +15,153 @@ related:
...
+  --header "Authorization: Token YourAuthToken" \
+  --header "Content-type: application/json"
+```
+
+##### Filter DBRP mappings by database
+```sh
+curl --request GET \
+  https://cloud2.influxdata.com/api/v2/dbrps?organization_id=example-org-id&database=example-db \
+  --header "Authorization: Token YourAuthToken" \
+  --header "Content-type: application/json"
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:65]: do we need content-type... json being there's no request body?

File: content/influxdb/cloud/query-data/influxql.md [hunk 8]
```diff
@@ -15,4 +15,153 @@ related:
...
+```
+
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:67]: Shall we add example to look up a specific DBRP id/bucket?

File: content/influxdb/cloud/query-data/influxql.md [hunk 9]
```diff
@@ -15,4 +15,153 @@ related:
...
+If you **do not find a DBRP mapping for a bucket**, complete the next procedure to map the unmapped bucket.
+_For more information on the DBRP mapping API, see the [`/api/v2/dbrps` endpoint documentation](/influxdb/cloud/api/#tag/DBRPs)._
+
+## Map unmapped buckets
+Use the [`/api/v2/dbrps` API endpoint](/influxdb/cloud/api/#operation/PostDBRP) to create a new DBRP mapping.
```
Comment (kelseiv) [inline content/influxdb/cloud/query-data/influxql.md:72]: mapping for a bucket.

File: content/influxdb/v2.0/query-data/influxql.md [hunk 10]
```diff
@@ -51,48 +89,100 @@ curl --request GET \
...
+Use the [`influx v1 dbrp create` command](/influxdb/v2.0/reference/cli/influx/v1/dbrp/create/)
+To map an unmapped bucket to a database and retention policy.
+Include the following:
 
- - organization (`organization` or `organization_id`)
- - target bucket (`bucket_id`)
- - database and retention policy to map to bucket (`database` and `retention_policy`)
+- **Database name** to map <span class="req">Required</span>
+- **Retention policy** name to map <span class="req">Required</span>
+- [Bucket ID](/influxdb/v2.0/organizations/buckets/view-buckets/#view-buckets-in-the-influxdb-ui) to map to <span class="req">Required</span>
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/query-data/influxql.md]: I'm going to create a shortcode for this.

File: content/influxdb/v2.0/query-data/influxql.md [hunk 11]
```diff
@@ -51,48 +89,100 @@ curl --request GET \
...
+
+- **Request method:** `POST`
+- **Headers:**
+  - **Authorization:** `Token` schema with your InfluxDB [autorization token](/influxdb/v2.0/security/tokens/)
+  - **Content-type:** `application/json`
+- **Request body:** JSON object with the following fields:
+  - **bucketID:** [bucket ID](/influxdb/v2.0/organizations/buckets/view-buckets/) <span class="req">Required</span>
+  - **database:** database name <span class="req">Required</span>
+  - **default:** set DBRP mapping to default
+  - **org** or **orgID:** organization name or [organization ID](/influxdb/v2.0/organizations/view-orgs/#view-your-organization-id) <span class="req">Required</span>
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/query-data/influxql.md]: Agree 😄

**Pull request merged**